inc/template-functions.php

```diff
@@ -337,10 +337,12 @@ function hale_filter_add_index_for_h2_elements( $content ) {
 function hale_get_ordered_content($content, $numbered_headings) {
 	$index = [];
 	$count = 0; //index number
-	$dom = new DomDocument();
+	$dom = new DOMDocument();
+	libxml_use_internal_errors(true);
```

Comment:
Maybe add in mb_convert_encoding()? ie  `$content = mb_convert_encoding($content, 'HTML-ENTITIES', 'UTF-8');`

Reply:
This is deprecated. 😭